content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/package.rst

```diff
@@ -19,22 +18,19 @@ Packages are commonly used for the following purposes:
 *Package use* is a field on the package form in Odoo that is only visible by enabling the *Batch
 Transfers* feature in settings of the *Inventory* application.
```

Comment:
Should we include a quick instruction on how to enable that, if the reader doesn't have it enabled already?